src/ILVerification/src/ILVerification.csproj

```diff
@@ -7,8 +7,12 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <TargetFramework>netstandard1.3</TargetFramework>
-    <AssemblyKey></AssemblyKey>
     <AssemblyOriginatorKeyFile>..\StrongNameKeys\ILVerify.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <!-- Suppress signing done by buildtools sign.targets -->
+    <SkipSigning>true</SkipSigning>
```

Comment:
Does this disable Authenticode signing? We still need that.

Reply:
We do not have the signing step in the CoreRT build pipeline at all. The CoreRT builds on myget are not signed.

This is a workaround to unblock Roslyn team. @sergiy-k and @MeiChin-Tsai will discuss more permanent solution.